effectiveOfficeBackend/src/main/kotlin/office/effective/features/user/routes/AuthRouting.kt

```diff
@@ -32,7 +34,12 @@ fun Route.authRoutingFun() {
         get("/callback") {
             try {
                 val principal: OAuthAccessTokenResponse.OAuth2? = call.principal()
```

Comment:
Maybe you should throw an exception if principal is null?

Reply:
Done